### DISCUSSION TITLE
Conflicting Documentation

### DISCUSSION THREAD

**Discussion by darinwil** [ContainerSSH/ContainerSSH#483] [Questions] [answered]
I am trying to figure out what to put in the Kubernetes section of the config webhook response and am having a little trouble determining what it should look like. According to the [OpenAPI](https://containerssh.io/reference/api/authconfig/#operation/getUserConfiguration) page there doesn't appear to be a "connection" parameter, however on the [reference manual](https://containerssh.io/reference/kubernetes/#configuring-connection-parameters) section it shows:
`kubernetes:`
`  connection:`
`    host: <...>`
`    <...>`
`
Which I only see under the kuberun backend on the OpenAPI page. Which page is correct and does someone have a full example config section for Kubernetes I could use for reference?

**Comment by ghost** [comment, marked as answer]
Hey there,

We recently discovered a bug in the OpenAPI generation process in 0.4.1, so our current OpenAPI documentation is, unfortunately, incorrect and we didn't have time to fix it yet. We are closing in on the 0.5 release which will fix this.

In the mean time, you can use the reference manual. The spec part of the container is equivalent to the Kubernetes PodSpec. You can use the kubectl explain command to explore the data structure.

If you need further help, please feel free to join the ContainerSSH Slack for debugging help.

**Comment by darinwil** [comment]
Perfect thank you!
